spec/features/register_spec.rb

```diff
@@ -22,12 +22,31 @@
       expect(page).to have_button('Register')
     end
 
-    scenario 'are redirected to user dashboard upon succesful registration' do
+    scenario 'they are redirected to user dashboard upon succesful registration' do
       fill_in 'Name', with: 'Lord Russell'
       fill_in 'Email', with: 'Russell@gmail.com'
       click_button 'Register'
 
       expect(page).to have_content("Lord Russell's Dashboard")
     end
+
+    scenario 'they are redirected back to registration view if registration fails' do
+      fill_in 'Name', with: ''
+      fill_in 'Email', with: ''
+      click_button 'Register'
+
+      expect(current_path).to eq("/register")
+      expect(page).to have_content('User Not Created: Required info missing.')
+    end
+
+    scenario 'they are redirected back to registration view if email is already taken' do
+      User.create!(name: 'Bob', email: 'russell@gmail.com')
```

Comment:
I like the way you created a user first. I would have just run the same input pattern twice. I like your way better.